README.md

```diff
@@ -69,7 +69,7 @@ wizAssets.getFileURI("img/ui/logo.jpg", successCallback, failCallback);
 
 **wizAssets.getFileURIs(Function success, Function fail);**
 
-- A success returns URI hashmap such as
+- A success returns a file URI hashmap such as
```

Comment:
Wouldn't it be better now if it returned an array of URIs? Does the full path have any meaning to the user here?